codefresh/provider.go

```diff
@@ -27,10 +35,9 @@ func Provider() *schema.Provider {
 }
 
 func configureProvider(d *schema.ResourceData) (interface{}, error) {
-	config := Config{
-		APIServer: d.Get("api_url").(string),
-		Token:     d.Get("token").(string),
-	}
 
-	return &config, nil
+	apiURL := d.Get("api_url").(string)
+	token := d.Get("token").(string)
```

Comment:
why we use "d" as variable name?

Reply:
it is a common pattern for `*schema.ResourceData` struct
https://www.terraform.io/docs/extend/writing-custom-providers.html#implement-create